src/web/mjs/connectors/CrunchyManga.mjs

```diff
@@ -88,9 +90,36 @@ export default class CrunchyManga extends Crunchyroll {
     _decryptImage(encrypted, key) {
         // create a view for the buffer
         let decrypted = new Uint8Array(encrypted);
-        for(let index in decrypted) {
+        for (let index in decrypted) {
             decrypted[index] ^= key;
         }
         return decrypted;
     }
+
+    async _getMangaFromURI(uri) {
+        await this._login();
+        const request = new Request(uri, this.requestOptions);
+        let response = await this.fetchDOM(request);
+        let responseData = await fetch(request);
+        let responseDataText = await responseData.text();
+
+        let title;
+        let seriesId;
+        let seriesIdList = [...responseDataText.matchAll(/series_id:(\d+)/g)];
+
+        if (seriesIdList.length > 0) {
+            title = response.querySelector('title').innerText.replace('Crunchyroll - ', '');
```

Comment:
```js
response.querySelector('div.queue-wrapper > a').title.trim()
```  
is probably better than a query and a replace
using a  .trim() is best practice